packages/terra-paginator/src/ControlledProgressivePaginator.jsx

```diff
@@ -68,27 +68,32 @@ class ProgressivePaginator extends React.Component {
 
   defaultProgressivePaginator() {
     const theme = this.context;
-    const totalPages = calculatePages(this.props.totalCount, this.props.itemCountPerPage);
+    const totalPages = (this.props.totalCount) ? calculatePages(this.props.totalCount, this.props.itemCountPerPage) : 0;
     const { selectedPage, intl } = this.props;
     const previousPageIndex = selectedPage === 1 ? 1 : selectedPage - 1;
     const nextPageIndex = selectedPage === totalPages ? totalPages : selectedPage + 1;
 
     return (
       <div className={cx('paginator', 'progressive', theme.className)} role="navigation" aria-label="pagination">
         <div>
-          {intl.formatMessage({ id: 'Terra.paginator.pageCount' }, { pageNumber: selectedPage, pageNumberTotal: totalPages })}
+          {(this.props.totalCount) ? intl.formatMessage({ id: 'Terra.paginator.pageCount' }, { pageNumber: selectedPage, pageNumberTotal: totalPages })
+            : intl.formatMessage({ id: 'Terra.paginator.pageIndex' }, { pageNumber: selectedPage })}
         </div>
         <div>
-          <PaginatorButton
-            ariaDisabled={selectedPage === 1}
-            ariaLabel={intl.formatMessage({ id: 'Terra.paginator.first' })}
-            className={cx(['nav-link', selectedPage === 1 ? 'is-disabled' : null])}
-            disabled={selectedPage === 1}
-            onClick={this.handlePageChange(1)}
-            onKeyDown={this.handlePageChange(1)}
-          >
-            {intl.formatMessage({ id: 'Terra.paginator.first' })}
-          </PaginatorButton>
+          {
+            this.props.totalCount && (
```

Comment:
Even if a total count wasn't provided, shouldn't the user always be able to return to the first page?

Reply:
we might need input from **UX** on this to display only `First` button with `previous` and `next`. below are the few screenshots which shows UI with only `First`.

<img width="1312" alt="Screen Shot 2020-11-05 at 3 42 39 PM" src="https://user-images.githubusercontent.com/12869809/98227980-fc590e80-1f7d-11eb-8652-e952b1ce2e91.png">
<img width="353" alt="Screen Shot 2020-11-05 at 3 43 56 PM" src="https://user-images.githubusercontent.com/12869809/98227987-ff53ff00-1f7d-11eb-94cd-b921bb785cc7.png">

Reply:
@neilpfeiffer mentioned in scrum that this showing first when total count is unknown is off balanced and not the designs for this, even though it could be coded. Looks good given the input. 👍

Reply:
Jump to the `first` page for this style/version of the paginator (with the total number of pages unknown) is not part of the design requirements created by UX and does not need to be included. 

The requested designs are shown on the issue description and do not need to be adjusted:
https://github.com/cerner/terra-core/issues/3157#issue-699929396